python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -626,6 +642,34 @@ def test_write_batch_pickle_duplicate_symbols(arctic_library):
     assert not lib.list_symbols()
 
 
+@pytest.mark.parametrize("num_columns", [8, 16])
+@pytest.mark.parametrize("num_days", [10, 200])
+@pytest.mark.parametrize("num_symbols", [100, 200])
+def test_write_batch_stress(arctic_library, num_columns, num_days, num_symbols):
```

Comment:
Shouldn't this go in the stress test module?

Reply:
we will change the name of this tests

Reply:
I would like it to be in this file, since it is the external API